### PR TITLE
To fix a few formatting typos in `Gradio-and-Wandb-Integration.md` tutorial

### PR DESCRIPTION
**Pull request by sadra-barikbin**
## Description
Hi there!

There was some formatting typos in the tutorial. This PR fixes them.

### CODE REVIEW

File: guides/09_other-tutorials/Gradio-and-Wandb-Integration.md [hunk 1]
```diff
@@ -49,10 +49,10 @@ Let's get started!
 
    Install Gradio and Wandb at the top:
 
-```sh
-
-pip install gradio wandb
-```
+   ```sh
+
```
Comment (aliabd) [inline guides/09_other-tutorials/Gradio-and-Wandb-Integration.md:53]:
```suggestion
```

File: guides/09_other-tutorials/Gradio-and-Wandb-Integration.md [hunk 2]
```diff
@@ -133,198 +133,118 @@ pip install gradio wandb
...
+   
+   generator = deepcopy(original_generator)
+   
+   ckpt = torch.load("/content/JoJoGAN/your-model-name.pt", map_location=lambda storage, loc: storage)
+   generator.load_state_dict(ckpt["g"], strict=False)
+   generator.eval()
+   
+   plt.rcParams['figure.dpi'] = 150
+   
+
```
Comment (aliabd) [inline guides/09_other-tutorials/Gradio-and-Wandb-Integration.md:188]:
```suggestion
```

File: guides/09_other-tutorials/Gradio-and-Wandb-Integration.md [hunk 3]
```diff
@@ -133,198 +133,118 @@ pip install gradio wandb
...
+   
+   
+   transform = transforms.Compose(
+       [
+           transforms.Resize((1024, 1024)),
+           transforms.ToTensor(),
+           transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5)),
+       ]
+   )
+
```
Comment (aliabd) [inline guides/09_other-tutorials/Gradio-and-Wandb-Integration.md:197]:
```suggestion
```

File: guides/09_other-tutorials/Gradio-and-Wandb-Integration.md [hunk 4]
```diff
@@ -133,198 +133,118 @@ pip install gradio wandb
...
+   
+   def inference(img):
+       img.save('out.jpg')
+       aligned_face = align_face('out.jpg')
+   
+       my_w = e4e_projection(aligned_face, "out.pt", device).unsqueeze(0)
+       with torch.no_grad():
+           my_sample = generator(my_w, input_is_latent=True)
+   
+
```
Comment (aliabd) [inline guides/09_other-tutorials/Gradio-and-Wandb-Integration.md:207]:
```suggestion
```

File: guides/09_other-tutorials/Gradio-and-Wandb-Integration.md [hunk 5]
```diff
@@ -133,198 +133,118 @@ pip install gradio wandb
...
-    gr.Image(type="pil"),
-    gr.Image(type="file"),
-    title=title,
-    description=description
-)
-
-demo.launch(share=True)
-```
+   ```python
+
```
Comment (aliabd) [inline guides/09_other-tutorials/Gradio-and-Wandb-Integration.md:216]:
```suggestion
```

File: guides/09_other-tutorials/Gradio-and-Wandb-Integration.md [hunk 6]
```diff
@@ -333,20 +253,20 @@ demo.integrate(wandb=wandb)
...
-    return gr.HTML(iframe)
-
-with gr.Blocks() as demo:
-    report_url = 'https://wandb.ai/_scott/pytorch-sweeps-demo/reports/loss-22-10-07-16-00-17---VmlldzoyNzU2NzAx'
-    report = wandb_report(report_url)
-
-demo.launch(share=True)
-```
+   ```python
+
```
Comment (aliabd) [inline guides/09_other-tutorials/Gradio-and-Wandb-Integration.md:257]:
```suggestion
```

**Review by aliabd — COMMENTED** [review]
Nice, thanks @sadra-barikbin! 

There are still some issues that are breaking the rendering. So i left some suggestions. But there is one important one. For some reason double empty lines can cause the codeblock to misbehave. Can you remove the following lines? 

- https://github.com/gradio-app/gradio/blob/0653c88e69db69096757a70f2becb97ba56c80f9/guides/09_other-tutorials/Gradio-and-Wandb-Integration.md?plain=1#L62
- https://github.com/gradio-app/gradio/blob/0653c88e69db69096757a70f2becb97ba56c80f9/guides/09_other-tutorials/Gradio-and-Wandb-Integration.md?plain=1#L69
- https://github.com/gradio-app/gradio/blob/0653c88e69db69096757a70f2becb97ba56c80f9/guides/09_other-tutorials/Gradio-and-Wandb-Integration.md?plain=1#L111

By the way, you can see a preview of the guide by visiting the website link at the gradio-pr-bot [comment](https://github.com/gradio-app/gradio/pull/8563#issuecomment-2171584809), just make sure to toggle to 'main' on the guide page to see the correct version.

**Review by pngwn — APPROVED** [review]
I've applied @aliabd's suggestions and will merge when CI runs. Thanks so much @sadra-barikbin!
